OxfordBodleian/Ullendorff/BDLeu2.xml

```diff
@@ -155,17 +155,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
                                 <date notBefore="1954" notAfter="1955"/>
-                                <persName ref="PRS14000WaldaA" role="scribe"/>
-                                <!--      <persName ref="PRS14000WaldaA" role="scribe"/> is indicated as the scribe in the final supplication on <locus target="#39v"/>. -->
-                                
+                              <persName ref="PRS14000WaldaA" role="scribe"/> is indicated as the scribe in the final supplication on <locus target="#39v"/>.                                
                                 <seg type="ink">Black, red.</seg>
+                                <desc>Modern bulky script, somewhat irregular.</desc>
                                 <seg type="rubrication">Incipit (two times two red lines alternating with black lines); holy names; names of the persons involved in
                                     the manuscript production;
                                     the words <foreign xml:lang="gez">ተዘከር፡</foreign> and <foreign xml:lang="gez">አምሕለከ፡</foreign>
                                     and <foreign xml:lang="gez">መሐልኩ፡ ለኪ፡</foreign> in the main text;
-                                    elements of numerals and punctuation marks.</seg>
-                                <desc>Modern bulky script, somewhat irregular.</desc>
-                                <!-- Order is ink, date, desc, rubr -->
+                                    elements of numerals and punctuation marks.</seg>                          
```

Comment:
"Punctuation marks" or "punctuation signs" – now we have both, but it could be standardised (if we want to – this could also easily be done at a later stage)

Reply:
If you think they need be standardised, I can also replace all of them now. I don't think the coexistence of the two could be misleading for the reader, but I understand that we might reach a higher level of standardisation.